proposals/mips/mip-b44/MIP-B44.md

```diff
@@ -0,0 +1,99 @@
+### **MIP-B44: Renewal of Moonwell Vaults Incentives Program**
+
+**Author(s):** 0xMims, BlockAnalitica & B.Protocol **Related Discussions:**
+[MIP-X08](https://moonwell.fi/governance/proposal/moonbeam?id=149) **Related
+Previous Proposal:** MIP-X08: Renewal of Moonwell Vaults Incentives Program and
+Frontier cbBTC Vault Ownership Transfer
+
+### **Summary**
+
+This proposal seeks to continue the successful Moonwell Morpho Vaults incentives
+program for an additional 90-day period. The Moonwell Flagship and Frontier
+vaults have consistently demonstrated strong performance and have become a
+cornerstone of Moonwell's lending ecosystem on Base. To maintain momentum and
+ensure continued competitive rewards for vault depositors, this proposal
+requests a grant of 10M WELL tokens.
+
+Following the recommendations from BlockAnalitica, the grant will be allocated
+across the following four vaults to optimize liquidity and address market
+demand:
+
+- **USDC:** 50%
+- **ETH:** 24%
+- **EURC:** 13%
+- **cbBTC:** 13%
+
+The program is scheduled to commence on or around Tuesday, June 17th, to
+minimize any gap in rewards for liquidity providers. The 10M WELL tokens will be
+transferred from the Foundation's multisig on Base to the Morpho URD to fund
+this 90-day incentive period.
+
+### **Success of the Ongoing Program**
+
+The Moonwell Morpho Vaults have established themselves as the dominant market
+leader on Morpho's Base deployment, consistently attracting significant Total
+Value Locked (TVL) and a large, active user base. The program's achievements
+underscore the community's strong appetite for optimized lending solutions and
+the effectiveness of the strategic incentives provided.
+
+The continued growth and high levels of adoption across the USDC, ETH, EURC, and
+cbBTC vaults validate the success of this initiative. The dynamic allocation
+model and robust risk management have provided risk-adjusted returns for
+suppliers and competitive rates for borrowers, solidifying Moonwell's leadership
+position within the Base ecosystem. Renewing these incentives is crucial to
+sustain this performance and continue fostering a vibrant and liquid lending
+market.
+
+### **Current Vault TVL**
+
+- As of JUne 2025, the vaults maintain a significant amount of Morpho TVL:
```

Comment:
This is a minor capitalization issue, can just correct it to be June instead of JUne